controllers/invites.ts

```diff
@@ -0,0 +1,95 @@
+import { addInviteToInviteModel, getUserDiscordInvite } from "../models/discordactions";
+import { InviteBodyRequest } from "../types/invites";
+import {CustomResponse} from "../types/global";
+import { addLog } from "../models/logs";
+import { generateDiscordInviteLink } from "../utils/discord-actions";
+import { verifyAuthToken } from "../utils/verifyAuthToken";
+
+export const createInviteLink = async (req: InviteBodyRequest, res: CustomResponse) => {
+  try {
+    const { userId, purpose } = req.body;
+    const authHeader = req.headers?.authorization;
+    if (!authHeader) {
+      return res.boom.unauthorized();
+    }
+    const token = authHeader.split(" ")[1];
```

Comment:
- What should be `authHeader.split(" ")[1];`?
- What if I don't pass space in header eg: `authorization: sdafklhjasdjkfh`;

Reply:
If you don't include a space after "Bearer," it will blend with the token, resulting in an invalid token and triggering an "Unauthorized" error.

Reply:
I also added this steps in implementation doc

Reply:
Where are we verifying "Bearer"?

Reply:
"Bearer" it should be any string not exactly

Reply:
Didn't get you.

Reply:
What you want?